### PR TITLE
Resurrect trillian createdb

### PR DESCRIPTION
**Pull request by k4leung4**
Signed-off-by: Kenny Leung <kleung@chainguard.dev>

#### Summary

This is to support the use case of using GCP Cloud SQL.
The database schema needs to be applied to the target MySQL instance.
My plan is to update the Trillian Helm chart to optionally create this batch job.

@vaikas @cpanato please take a look.

#### Ticket Link

Fixes

#### Release Note

```release-note

```

### CODE REVIEW

File: config/trillian/createdb/101-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
```
Comment (cpanato) [inline config/trillian/createdb/101-secret.yaml:2]: same here as well

File: config/trillian/createdb/101-service-account.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Copyright 2021 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
```
Comment (cpanato) [inline config/trillian/createdb/101-service-account.yaml:2]: here too :)

**Review by cpanato — COMMENTED** [review]
some very small non code comments, otherwise lgtm
thanks for doing this

File: cmd/trillian/createdb/main.go [hunk 3]
```diff
@@ -0,0 +1,296 @@
+/*
+Copyright 2021 Chainguard, Inc.
+SPDX-License-Identifier: Apache-2.0
+*/
```
[the remark this answers is not in the document]
Comment (k4leung4) [reply cmd/trillian/createdb/main.go:4]: fixed.

File: config/trillian/createdb/100-namespace.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Copyright 2021 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
```
[the remark this answers is not in the document]
Comment (k4leung4) [reply config/trillian/createdb/100-namespace.yaml:2]: removed.

File: config/trillian/createdb/101-secret.yaml (hunk 1 above)
Comment (k4leung4) [reply config/trillian/createdb/101-secret.yaml:2]: removed.

File: config/trillian/createdb/placeholder.go [hunk 5]
```diff
@@ -0,0 +1,6 @@
+/*
+Copyright 2021 Chainguard, Inc.
+SPDX-License-Identifier: Apache-2.0
```
[the remark this answers is not in the document]
Comment (k4leung4) [reply config/trillian/createdb/placeholder.go:3]: fixed.

**Comment by k4leung4** [comment]
> some very small non code comments, otherwise lgtm thanks for doing this

thanks for the review.
please take another look.

**Review by cpanato — APPROVED** [review]
thank you for working on this!
super nice

**Review by vaikas — APPROVED** [review]
Thanks!!!

**Pull request merged**